pkgs/applications/emulators/craftos-pc/default.nix

```diff
@@ -28,6 +28,12 @@ let
     rev = "v${version}";
     sha256 = "sha256-BXTsBMlsymQHABWQCiv22Ia5jm2xv1jNy7Unwymtyp0=";
   };
+  # TODO: craftos2-lua supports bsd, freebsd, mingw, posix as well
```

Comment:
If it's helpful, there's a full list of platforms here:

https://github.com/NixOS/nixpkgs/blob/193374f9756ea3d3bd91d38fdb7124c1104957c7/lib/systems/doubles.nix#L95